iXBRLViewerPlugin/viewer/src/js/util.js

```diff
@@ -207,3 +210,20 @@ export function getIXHiddenLinkStyle(domNode) {
     }
     return null;
 }
+
+/**
+ * Transforms measure qname into title case label (or currency symbol, if applicable).
+ * @return {String} Measure Label
+ */
+
+export function measureLabel(report, measure) {
+    const qname = report.qname(measure);
+    if (qname.namespace === NAMESPACE_ISO4217) {
+        measure = i18next.t(`currencies:unitFormat${qname.localname}`, {defaultValue: qname.localname});
+    } else if (measure.includes(':')) {
+        measure = titleCase(measure.split(':')[1]);
```

Comment:
I actually quite like the idea of including the prefixes for custom units so it'd be clear if a preparer has created their own units, but a) this never happens in practice b) we should treat other namespaces as "standard" (`xbrli` and `utr`) so I think on balance it is better to remove all prefixes for now, as per this change.

There's been some discussion about making more use of UTR definitions in the viewer, so we could perhaps do it as part of that.  In theory, UTR validation prevents misuse of custom units in most cases.

Reply:
I was going down this line of thinking as well. If we established a mapping of `utr:XYZ` to symbols, we would have the opportunity for improved control of unit display (e.g. `m³` instead of `m3`). This mapping could be static, or it could be dynamically loaded by Arelle, which is already building UTR entries details (with a couple of changes, UTR `definition` could be brought over as well to be used in a tooltip or unit legend).

For any custom units that wouldn't have a match in this mapping, we could render the full `prefix:unit` display for clarity (however rarely custom units are used).

Reply:
I just saw #576, so I anticipate this will all be addressed with that solution.